acmwebsite/public/css/style.css

```diff
@@ -108,3 +108,8 @@
 .checkbox-group .checkbox label {
   font-weight: normal;
 }
+
+/* Schedule Page */
+.acm-meeting-title {
```

Comment:
Not an issue now, but I want this web system to eventually be extensible to other clubs. Naming CSS classes like this is stupid and will make that project harder.

Reply:
Ahh, I agree. I think that was a leftover from the s/lug/acm/g that we did in about 10 minutes :joy: